_tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md

```diff
@@ -24,9 +21,9 @@ Rolling upgrades, sometimes referred to as *node replacement upgrades*, can be p
 
 Review [Upgrading OpenSearch]({{site.url}}{{site.baseurl}}/upgrade-opensearch/index/) for recommendations about backing up your configuration files and creating a snapshot of the cluster state and indexes before you make any changes to your OpenSearch cluster.
 
-Before migrating to remote-backed storage, upgrade to OpenSearch 2.14.
+Before migrating to remote-backed storage, upgrade to OpenSearch 2.15.
```

Comment:
```suggestion
Before migrating to remote-backed storage, upgrade to the latest version of OpenSearch.
```

Reply:
They need to minimally upgrade to 2.15 for migration , latest versions is not mandatory .